### PR TITLE
Lpd 18035

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://liferay.atlassian.net/browse/LPD-18419

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/editor/configuration/CommerceCKEditorConfigContributor.java [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+		}
+
+		EditorConfigContributor ckEditorConfigContributor =
+			_ckEditorConfigContributorSnapshot.get();
+
+		ckEditorConfigContributor.populateConfigJSONObject(
+			jsonObject, inputEditorTaglibAttributes, themeDisplay,
+			requestBackedPortletURLFactory);
+
+		String extraPlugins = (String)jsonObject.get("extraPlugins");
```
Comment (smottal) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/editor/configuration/CommerceCKEditorConfigContributor.java:66]: Can this be moved inside the put where is used?
```
put("extraPlugin", () -> { String extraPlugins = (String)jsonObject.get("extraPlugins"); return extraPlugins.concat(",aicreator"); }
```

**Review by smottal — APPROVED** [review]
Just a little style change but not mandatory.

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
I've asked for a review here https://github.com/liferay-content-management/liferay-portal/pull/5252

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**